composer.json

```diff
@@ -6,36 +6,49 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.4",
-        "drupal/core": "^9.3",
+        "php": ">=8.0",
+        "drupal/core": "^9.4",
         "drupal/language_selection_page": "^2.10",
         "drupal/pathauto": "^1.8"
     },
     "require-dev": {
         "behat/behat": "^3.10",
         "composer/installers": "~1.5",
         "drupal/config_devel": "~1.2",
-        "drupal/core-composer-scaffold": "^9.3",
-        "drupal/core-dev": "^9.3",
-        "drupal/ctools": "3.9.0",
+        "drupal/core-composer-scaffold": "^9.4",
+        "drupal/ctools": "^4.0",
         "drupal/devel": "~4.1",
-        "drupal/drupal-extension": "~4.0",
+        "drupal/drupal-extension": "dev-master",
         "drupal/redirect": "^1.6",
         "drupal/token": "^1.10",
         "drush/drush": "~10.3",
-        "fabpot/goutte": "^3.3",
+        "guzzlehttp/guzzle": "^7.0",
+        "masterminds/html5": "^2.7.6",
+        "mikey179/vfsstream": "^1.6.10",
         "openeuropa/behat-transformation-context": "~0.1",
         "openeuropa/code-review": "^2.0",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta5",
         "phpspec/prophecy-phpunit": "^2",
-        "symfony/dom-crawler": "^4.4.12"
+        "symfony/browser-kit": "^4.4.30",
+        "symfony/css-selector": "^4.4.23",
+        "symfony/dom-crawler": "^4.4.44",
+        "symfony/event-dispatcher": "^4.4.28",
+        "symfony/http-foundation": "^4.4.30",
+        "symfony/phpunit-bridge": "^6.0",
+        "symfony/routing": "^4.4.28",
+        "symfony/validator": "^4.4.30",
+        "symfony/yaml": "^4.4.28"
```

Comment:
Probably we may set actual dependencies:
![image](https://user-images.githubusercontent.com/9432036/204018446-a2136bf7-d83c-4893-8c3e-a646d16cc1ac.png)
```suggestion
        "symfony/browser-kit": "^4.4.37",
        "symfony/css-selector": "^4.4.24",
        "symfony/dom-crawler": "^4.4.44",
        "symfony/event-dispatcher": "^4.4.30",
        "symfony/http-foundation": "^4.4.30",
        "symfony/phpunit-bridge": "^6.0",
        "symfony/routing": "^4.4.30",
        "symfony/validator": "^4.4.30",
        "symfony/yaml": "^4.4.29"
```
Did checking by run of composer with ` --prefer-lowest --prefer-stable` options.

Reply:
The proposed versions wouldn't be the lowest compatible versions.
Why would we increase the lowest supported version if the current ones are compatibile?